rosa_getting_started/rosa-quickstart-guide-ui.adoc

```diff
@@ -1,17 +1,17 @@
 :_mod-docs-content-type: ASSEMBLY
 [id="rosa-quickstart-guide-ui"]
-= {product-title} quickstart guide
+= {product-title} quick start guide
 include::_attributes/attributes-openshift-dedicated.adoc[]
 :context: rosa-quickstart-guide-ui
 
 toc::[]
 
 [NOTE]
 ====
-If you are looking for a comprehensive getting started guide for ROSA, see xref:../rosa_getting_started/rosa-getting-started.adoc#rosa-getting-started[Comprehensive guide to getting started with {product-title}]. For additional information on ROSA installation, see link:https://www.redhat.com/en/products/interactive-walkthrough/install-rosa[Installing Red Hat OpenShift Service on AWS (ROSA) interactive walkthrough].
+If you are looking for a comprehensive getting started guide for {product-rosa} (ROSA), see xref:../rosa_getting_started/rosa-getting-started.adoc#rosa-getting-started[Comprehensive guide to getting started with {product-title}]. For additional information on ROSA installation, see link:https://www.redhat.com/en/products/interactive-walkthrough/install-rosa[Installing Red Hat OpenShift Service on AWS (ROSA) interactive walkthrough].
```

Comment:
I think `{product-rosa}` is missing from the common attributes. Should it be `{product-title}`?
![Screenshot 2024-02-16 at 2 59 34 PM](https://github.com/openshift/openshift-docs/assets/96192119/b280e126-3468-42ec-b451-82108dbcc9fd)

Reply:
Good catch-thanks @kcarmichael08!